packages/dds/merge-tree/src/mergeTree.ts

```diff
@@ -2975,35 +2766,18 @@ export class MergeTree {
         for (let childIndex = 0; childIndex < node.childCount; childIndex++) {
             const child = children[childIndex];
             const len = this.nodeLength(child, refSeq, clientId) ?? 0;
-            if (MergeTree.traceTraversal) {
-                let segInfo: string;
-                if ((!child.isLeaf()) && this.collabWindow.collaborating) {
-                    segInfo = `minLength: ${child.partialLengths!.minLength}`;
-                } else {
-                    const segment = <ISegment>child;
-                    segInfo = `cli: ${glc(this, segment.clientId)} seq: ${segment.seq} text: '${segment.toString()}'`;
-                    if (segment.removedSeq !== undefined) {
-                        segInfo += ` rcli: ${glc(this, segment.removedClientId!)} rseq: ${segment.removedSeq}`;
-                    }
-                }
-                // eslint-disable-next-line max-len
-                console.log(`@tcli ${glc(this, this.collabWindow.clientId)}: map len: ${len} start: ${_start} end: ${_end} ${segInfo}`);
-            }
             if (go && (_end > 0) && (len > 0) && (_start < len)) {
                 // Found entry containing pos
                 if (!child.isLeaf()) {
                     if (go) {
                         go = this.nodeMap(child, actions, _pos, refSeq, clientId, accum, _start, _end);
                     }
                 } else {
-                    if (MergeTree.traceTraversal) {
-                        console.log(`@tcli ${glc(this, this.collabWindow.clientId)}: map leaf action`);
-                    }
                     if (actions.leaf) {
```

Comment:
This could be else if.

Reply:
this is kind of intentional as this PR is wholly cleanup. i didn't make any other changes.